### DISCUSSION TITLE
Setting task priority by hashtag or similar in summary

### DISCUSSION THREAD

**Discussion by DerFlob** [TechbeeAT/jtxBoard#1563] [Ideas]
Hello, 

first off, thank you for your great work an the app! Much appreciated. :)

I was wondering, if it would be possible to set the task priority more easily. As far as I can see, currently you have to create a task, scroll down to show all fields, scroll back up tp the priority field and then set it.

Could there be a similar feature, to the hashtag being interpreted as a category for the priority of a task? Maybe using the exclamation mark and a priority name? I.e. `!highest` or `!low` to automatically set the prio.

Thank you!

**Comment by patrickunterwegs** [comment]
Hey @DerFlob !
Thank you for your proposal! Yes, let's think about that. 
When using something like you proposed, I wouldn't like to use the English word, but the priority name in the language of the app. I think that would be clearer. What do you think?

**Reply by DerFlob** [reply]
Hello Patrick, thank you for your quick reply! :)

I think it would make sense to localize the words for sure, but maybe also have the fallback to the English words?

**Comment by scoop96** [comment]
@patrickunterwegs 

Hello, spanish guy here
you could also come up with a "code" to set priority and avoid a word (Also would be quicker) 

for example if the new task first line starts by `<x>` where X is a number you setted the priority to that value 

and if you wanted to type a task with those values (weird but okay) you can avoid it by simply doing `"<3> my weird task "`

that way we all can be happy (i would love to have a quick way to set priority as simple as it is tagging)
